<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEbsdReferenceFrameDialog</class>
 <widget class="QDialog" name="QEbsdReferenceFrameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Reference Frame Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>8</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item row="1" column="1" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>IPF Reference Direction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="refDir">
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Z</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="degToRads">
     <property name="text">
      <string>Convert Displayed Data to Radians 
(Imported Data will NOT be changed)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="m_TSLdefaultBtn">
       <property name="text">
        <string>Edax - TSL (Default)</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_HKLdefaultBtn">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Oxford - HKL (Default)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_HEDMdefaultBtn">
       <property name="text">
        <string>HEDM - IceNine (Default)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_NoTransBtn">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>No/Unknown Transformation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>X Dim:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_XDim">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Y Dim:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_YDim">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>X Res:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_XRes">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Y Res:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_YRes">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="ipfLabel">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QEbsdReferenceFrameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QEbsdReferenceFrameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
